platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -65,7 +65,9 @@ public static synchronized Mapbox getInstance(@NonNull Context context, @NonNull
       }
 
       ConnectivityReceiver.instance(appContext);
+      Timber.d("instance: connectivity receiver started");
```

Comment:
How adding a log solves a JNI bug?
In any case, we should remove these logs before merging because they'd clutter Logcat unnecessarily.